src/screens/MemberDetail/MemberDetail.tsx

```
@@ -271,224 +274,473 @@
-                        }}
-                        src={sanitizeAvatars(
-                          selectedAvatar,
-                          formState.avatarURL,
+
+      {/* Tab Navigation */}
+      <div className={styles.tabNavigation}>
+        <button
+          className={`${styles.tabButton} ${activeTab === 'overview' ? styles.tabButtonActive : ''}`}
+          onClick={() => setActiveTab('overview')}
+          data-testid="overviewTab"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)